src/silo/common/fasta_reader.test.cpp

```diff
@@ -0,0 +1,71 @@
+#include <gtest/gtest.h>
+#include <string>
+#include <unordered_map>
+
+#include "silo/common/fasta_format_exception.h"
+#include "silo/common/fasta_reader.h"
+#include "silo/preprocessing/preprocessing_config.h"
+
+TEST(FastaReader, shouldReadFastaFile) {
+   const silo::InputDirectory input_directory{"./testBaseData/fastaFiles/"};
+   const silo::SequenceFilename sequence_filename{"test.fasta"};
+   const std::filesystem::path file_path =
+      silo::createPath(input_directory.directory, sequence_filename.filename);
+   silo::FastaReader in_file(file_path);
```

Comment:
Nitpicking: Strictly speaking, this is not really a file. I'd either call it `under_test` or `fasta_reader`.